privacy-config/privacy-config-impl/src/main/java/com/duckduckgo/privacy/config/impl/features/trackingparameters/TrackingParametersPlugin.kt

```diff
@@ -19,19 +19,21 @@ package com.duckduckgo.privacy.config.impl.features.trackingparameters
 import com.duckduckgo.di.scopes.AppScope
 import com.duckduckgo.feature.toggles.api.FeatureName
 import com.duckduckgo.privacy.config.api.PrivacyFeatureName
-import com.duckduckgo.privacy.config.impl.features.privacyFeatureValueOf
 import com.duckduckgo.privacy.config.api.PrivacyFeaturePlugin
+import com.duckduckgo.privacy.config.impl.features.privacyFeatureValueOf
+import com.duckduckgo.privacy.config.impl.version.VersionHandler
 import com.duckduckgo.privacy.config.store.*
 import com.duckduckgo.privacy.config.store.features.trackingparameters.TrackingParametersRepository
 import com.squareup.anvil.annotations.ContributesMultibinding
-import javax.inject.Inject
 import com.squareup.moshi.JsonAdapter
 import com.squareup.moshi.Moshi
+import javax.inject.Inject
 
 @ContributesMultibinding(AppScope::class)
 class TrackingParametersPlugin @Inject constructor(
     private val trackingParametersRepository: TrackingParametersRepository,
-    private val privacyFeatureTogglesRepository: PrivacyFeatureTogglesRepository
+    private val privacyFeatureTogglesRepository: PrivacyFeatureTogglesRepository,
+    private val versionHandler: VersionHandler
```

Comment:
We don't really need this new `VersionHandler` class. We can just use `AppBuildConfig`

Reply:
I've kept the `VersionHandler` so I don't have to repeat this everywhere:
```
if (minSupportedVersion == null) return true
return appBuildConfig.versionCode >= minSupportedVersion
```
I thought about adding that logic to `AppBuildConfig` but it doesn't feel like it makes sense to put it there.

Reply:
`VersionHandler` has been removed.